inference-engine/src/offline_transformations/src/pruning/init_const_mask.cpp

```diff
@@ -40,13 +40,15 @@ ngraph::pass::InitConstMask::InitConstMask(const ngraph::AxisSet & dims,
                 end[dim] = value + 1;
 
                 bool skip_dim_value = false;
+                NGRAPH_SUPPRESS_DEPRECATED_START
                 CoordinateTransform iter(shape, begin, end);
```

Comment:
who will do removing of its usage? do we have epic with tasks? what is the alternative?

Reply:
The alternative is to use knowledges about data structure.instead of this structure.
Moreover it is our internal structure and we deprecate it in order to don't use it in new reference implementations if it is not needed.